mne_bids/read.py

```diff
@@ -147,9 +147,13 @@ def _read_events(events_data, event_id, raw, verbose=None):
         verbose=verbose
     )
     all_dur = raw.annotations.duration
-    if all_events.size == 0:
-        warn('No events found or provided. Please add annotations to the raw '
-             'data, or provide the events_data and event_id parameters.')
+    if all_events.size == 0 and verbose:
+        # Confirm from filename dataset is not likely resting state
+        if '_task-rest' not in raw.filenames[0]:
```

Comment:
```suggestion
    if all_events.size == 0 and verbose:
        # Confirm from filename dataset is not likely resting state
        if '_task-rest' not in raw.filenames[0]:
```
Actually **this** needs to be fixed, not the tests per se.

```suggestion
if all_events.size == 0 and `_task_rest` not in raw.filenames[0]:
```